pkg/node/state_fsm/fsm_idle.go

```diff
@@ -19,6 +19,25 @@ type InvRequester interface {
 	Request(p types.MessageSender, id []byte)
 }
 
+var (
+	idleSkipMessageList = proto.PeerMessageIDs{
+		proto.ContentIDGetSignatures,
```

Comment:
GetSignatures, GetBlock, GetBlockIds messages should be allowed to enable blocks distribution from idling nodes.